apps/extension/src/wasm-build-action.ts

```diff
@@ -50,7 +51,7 @@ async function executeWorker(
   const actionPlanType = transactionPlan.actions[actionPlanIndex]?.action.case;
   if (!actionPlanType) throw new Error('No action key provided');
 
-  await penumbraWasmModule.loadProvingKey(actionPlanType);
+  await penumbraWasmModule.loadProvingKey(camelToSnakeCase(actionPlanType));
```

Comment:
This was the beast that took forever to solve 😩 Per #469, errors inside `ViewProtocolServer` impl's are being swallowed, so it took rebuilding the WASM crate, adding log lines everywhere, etc. to discover that we were trying to load the `swapClaim` proving key when we should have been trying to load the `swap_claim` key 🤦🏻‍♂️

Reply:
@jessepinho We can actually simply remove `{ keyType: 'nullifierDerivation', file: 'nullifier_derivation_pk.bin' }` and change convert to `{ keyType: 'undelegateClaim', file: 'convert_pk.bin' }`.

Reply:
@grod220 We're [calling out to WASM](https://github.com/penumbra-zone/web/blob/main/packages/wasm/src/utils.ts#L18) with the snake-cased key type; WASM then maps snake-cased keys to filenames [here](https://github.com/penumbra-zone/penumbra/blob/main/crates/wasm/src/keys.rs#L26-L45). So I figured it'd be best to keep the names consistent between the two. Thoughts?

Reply:
Ah, I see. Hmm. Kinda on the fence. Ideally, the type system could assist us in ensuring we don't accidentally put the wrong strings there. Given `load_proving_key` is made for the web, it feels ok to convert those to camelCase so that we can use the bufbuild types.

Reply:
This change will be easier to make after migrating to web repo

Reply:
Ah, forgot about this comment. But OK, will leave it as-is then.